fmriprep/workflows/bold/base.py

```diff
@@ -243,15 +246,19 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
             'magnitude2': 'sub-03/ses-2/fmap/sub-03_ses-2_run-1_magnitude2.nii.gz'
         }]
         run_stc = True
+        multiecho = False
         bold_pe = 'j'
     else:
-        if isinstance(bold_file, list):  # For multiecho data, grab TEs
+        if multiecho is True:  # For multiecho data, grab TEs
```

Comment:
could multiecho not be a boolean? If it cannot:

```
if multiecho:
   ...
```

Reply:
I could just keep using `isinstance(bold_file, list)` as a proxy, but since it's used four times I thought a boolean would be easier to interpret. If you'd prefer a non-boolean, though, I can stick to the `isinstance` solution !